### PR TITLE
Fix mint-radio breaking layout on opera mini

### PR DESCRIPTION
**Pull request by aju**
closes https://github.com/brainly/style-guide/issues/529

### CODE REVIEW

File: src/components/form-elements/_checkbox-radio.scss [hunk 1]
```diff
@@ -1,5 +1,4 @@
-$crInputSize: fontSize(default);
-$crInputFontSize: fontSize(default);
+$crInputSize: rhythm(2 / 3);
```
Comment (aju) [inline src/components/form-elements/_checkbox-radio.scss]: I think this is better as we get full units (sometimes units like 15.98px were breaking layout for me)

File: src/components/form-elements/_checkbox-radio.scss [hunk 2]
```diff
@@ -63,13 +61,10 @@ $includeHtml: false !default;
 
     &:before {
       content: '';
-      position: absolute;
```
Comment (aju) [inline src/components/form-elements/_checkbox-radio.scss]: position absolute inside flex element was making opera mini crazy

File: src/components/form-elements/_checkbox-radio.scss [hunk 3]
```diff
@@ -17,14 +16,13 @@ $includeHtml: false !default;
...
     min-height: $crInputSize;
-    font-size: $crInputFontSize;
 
     &__element {
       opacity: 0;
       position: absolute;
       margin: 0;
-      width: 100%;
-      height: 100%;
+      width: $crInputSize;
```
Comment (aju) [inline src/components/form-elements/_checkbox-radio.scss]: No more fluid values

**Comment by matzimowski** [comment]
:+1:

File: src/components/form-elements/_checkbox-radio.scss (hunk 1 above)
Comment (mr-mig) [inline src/components/form-elements/_checkbox-radio.scss]: We can make a shorthands, like `$third`

Comment (mr-mig) [inline src/components/form-elements/_checkbox-radio.scss]: In general, I am ok with any of that

**Comment by mr-mig** [comment]
:+1:

File: src/components/form-elements/_checkbox-radio.scss (hunk 1 above)
Comment (aju) [inline src/components/form-elements/_checkbox-radio.scss]: I will switch it to 0.667 to make it simpler to refactor in the future

**Pull request merged**